apps/meteor/client/views/admin/users/AdminUsersPage.tsx

```diff
@@ -68,18 +70,18 @@ const UsersPage = (): ReactElement => {
 						<Tabs.Item selected={!tab || tab === 'all'} onClick={() => setTab('all')}>
 							{t('All')}
 						</Tabs.Item>
-						<Tabs.Item selected={tab === 'invited'} onClick={() => setTab('invited')}>
-							{t('Invited')}
-						</Tabs.Item>
-						<Tabs.Item selected={tab === 'new'} onClick={() => setTab('new')}>
-							{t('New_users')}
+						<Tabs.Item selected={tab === 'pending'} onClick={() => setTab('pending')}>
+							{pendingActionsCount === 0 ? t('Pending') : `${t('Pending')} (${pendingActionsCount})`}
```

Comment:
pendingActionsCount is defined and used but since the set is never ran the number will never change. Why is it incomplete?